sacc/tracers.py

```diff
@@ -1,5 +1,7 @@
 import numpy as np
+import qp
```

Comment:
Since many users may not be using qp, could you move the two imports inside the method where it's used so it won't need to be imported unless used?

Reply:
Sure, but that won't really change much as I've also added qp to the requirements.   Do you want to remove it from there?

Reply:
I just meant to speed up the import time - it looks like it would save about 40% to the time to import the package in some simple tests. It's not much in absolute but on NERSC where the file systems can be dodgy it's often worth doing.

Reply:
Oh, interesting.  Which version of qp and tables_io?  I've put a fair bit of work to make those lighter recently.   Anyway, I'll move this so we can close this and move on.

Reply:
Looks like I tested with 0.8.1.

Reply:
Actually unresolving - could you move the tables_io one as well?